src/Common/QR/Notices.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * The Notices class for the QR module.
+ *
+ * @since TBD
+ *
+ * @package TEC\Common\QR
+ */
+
+namespace TEC\Common\QR;
+
+/**
+ * Class Notices
+ *
+ * @since   TBD
```

Comment:
```suggestion
 * @since TBD
```

Reply:
Replace in other places as well, not sure why the phpcs rule is not catching it. It should be flagged.